### PR TITLE
[NET 6 Attribute Conversion] MediaPlayer

### PR DESCRIPTION
**Pull request by stephen-hawley**

### CODE REVIEW

File: src/MediaPlayer/MPMoviePlayerController.cs [hunk 1]
```diff
@@ -19,8 +19,21 @@ public partial class MPMoviePlayerController {
...
+#if NET
+		[SupportedOSPlatform ("ios7.0")]
+		[SupportedOSPlatform ("maccatalyst14.0")]
+		[UnsupportedOSPlatform ("ios9.0")]
+#if IOS
+		[Obsolete ("Starting with ios9.0 use 'AVPlayerViewController' (AVKit) instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
+		[UnsupportedOSPlatform ("tvos")]
+#if IOS
+		[Obsolete ("Starting with ios15.0.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
```
Comment (tj-devel709) [inline src/MediaPlayer/MPMoviePlayerController.cs:31]: Double `#if IOS` looks a little strange

**Review by tj-devel709 — APPROVED** [review]

File: src/MediaPlayer/MPMoviePlayerController.cs (hunk 1 above)
Comment (rolfbjarne) [reply src/MediaPlayer/MPMoviePlayerController.cs:29]: I'm guessing one comes from the type, and another one from the the method?

**Review by rolfbjarne — APPROVED** [review]

**Comment by chamons** [comment]
I changed the PR title to match the pattern.

File: src/MediaPlayer/MPMoviePlayerController.cs (hunk 1 above)
Comment (chamons) [reply src/MediaPlayer/MPMoviePlayerController.cs:29]: Yep, here is the definition:

```
	[Deprecated (PlatformName.iOS, 9, 0, message: "Use 'AVPlayerViewController' (AVKit) instead.")]
	[MacCatalyst (14,0)] // docs says 13.0 but this throws: NSInvalidArgumentException Reason: MPMoviePlayerController is no longer available. Use AVPlayerViewController in AVKit.
	[BaseType (typeof (NSObject))]
	interface MPMoviePlayerController : MPMediaPlayback {
```

I'd just pick the oldest one but whatever.

**Review by chamons — APPROVED** [review]

**Comment by chamons** [comment]
👍  after fixing the double obsolete and getting green build.

**Pull request merged**